RELEASE.md

```diff
@@ -1,3 +1,24 @@
+# Release 1.1.0
+
+## New features
+
+- New field `numberOfVectors` in the response of the `get_stats` endpoint (https://github.com/marqo-ai/marqo/pull/553). 
+This field indicates the number of vectors in an index so that users can monitor the status of their index. See [here](https://docs.marqo.ai/1.1.0/API-Reference/stats/) for detailed usage.
+
+- New health check endpoint `client.index("your-index").health()` (https://github.com/marqo-ai/marqo/pull/552). 
+We move the health check under the index API to make it more intuitive to use. The old health check endpoint `client.health()` is deprecated and will be removed in Marqo 2.0.0. See [here](https://docs.marqo.ai/1.0.0/API-Reference/health/) for detailed usage.
```

Comment:
Wording:

"We move the health check under"
-> 

"This replaces old cluster-level health check"  
(Grammar + not using "we")